yt_dlp/extractor/joqrag.py

```diff
@@ -0,0 +1,54 @@
+import urllib.parse
+
+from .common import InfoExtractor
+
+
+class JoqrAgIE(InfoExtractor):
+    IE_DESC = '超!A&G+ 文化放送 Nippon Cultural Broadcasting, Inc. (JOQR)'
+    _VALID_URL = r'''(?x)
+                    (https?://www\.uniqueradio\.jp/agplayer5/player\.php)|
+                    (https?://www\.uniqueradio\.jp/agplayer5/inc-player-hls\.php)|
+                    (https?://(?:www\.)?joqr\.co\.jp/ag/)|
+                    (https?://(?:www\.)?joqr\.co\.jp/qr/(?:agdailyprogram|agregularprogram)/)
+                    '''
+    _TESTS = [{
+        'url': 'https://www.uniqueradio.jp/agplayer5/player.php',
+        'info_dict': {
+            'id': 'live',
+            'title': str,
+            'description': str,
+            'live_status': 'is_live',
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+    }]
```

Comment:
could you add some more tests for the other urls in `_VALID_URL`?
just something like this is fine, you don't need to put the same details in again and again:
```python3
    {
        'url': 'https://www.joqr.co.jp/ag/article/103760/',
        'only_matching': True,
    }, {
        'url': 'http://www.joqr.co.jp/qr/agregularprogram/',
        'only_matching': True,
    } # etc etc
```